# origin: Debian (herbert)
# cset: n/a
# inclusion: submitted (2004-09-18)
# description: set owner for radeonfb module, fix red/blue typo
# revision date: 2004-09-04

diff -urN kernel-source-2.4.26/drivers/video/radeonfb.c kernel-source-2.4.26-1/drivers/video/radeonfb.c
--- kernel-source-2.4.26/drivers/video/radeonfb.c	2004-04-14 23:05:39.000000000 +1000
+++ kernel-source-2.4.26-1/drivers/video/radeonfb.c	2004-04-17 14:24:02.000000000 +1000
@@ -935,6 +935,7 @@
 #endif /* CONFIG_PMAC_BACKLIGHT */
 
 static struct fb_ops radeon_fb_ops = {
+	owner:			THIS_MODULE,
 	fb_get_fix:		radeonfb_get_fix,
 	fb_get_var:		radeonfb_get_var,
 	fb_set_var:		radeonfb_set_var,
@@ -2368,7 +2369,7 @@
 			disp->visual = FB_VISUAL_DIRECTCOLOR;
 			v.red.offset = 10;
 			v.green.offset = 5;
-			v.red.offset = 0;
+			v.blue.offset = 0;
 			v.red.length = v.green.length = v.blue.length = 5;
 			v.transp.offset = v.transp.length = 0;
 			break;

